atlp-rwanda/e-comm-team-techsmith-fn
# #185218928 Users should be able to search

## What does this PR do?
Users should be able to search 

## Description of Task to be completed?
- Given that a user is logged in when the user enters data into the search component, then the records displayed in the table should reflect all possible records relevant to the content of the data entered by the user in the search component

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout <ft-search-product-#185218928>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?
[#185218928 ](https://www.pivotaltracker.com/story/show/185218928)

## Screenshots (if appropriate)


Uploading Screen Recording 2023-06-18 at 20.12.51.mov…



Uploading Screen Recording 2023-06-18 at 20.19.14.mov…
